client/src/components/LandingPage/index.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import { makeStyles } from "@material-ui/core";
+import { useHistory, Link } from "react-router-dom";
+
+import FocusLogo from "../../images/focus-logo.svg";
+import styles from "./style.module.scss";
+
+const useStyles = makeStyles({
+    root: {
+        "& .MuiOutlinedInput-root": {
+            borderRadius: 15,
+        },
+        "& .MuiInputLabel-root": {
+            fontSize: "1.25rem",
+            color: "#7D7D7D",
+            fontFamily: "'Ropa Sans', sans-serif",
+        },
+    },
+    buttonRoot: {
+        background: "#30A0F5",
+        borderRadius: 50,
+        padding: "4px 25px",
+        width: "100%",
+    },
+    label: {
+        textTransform: "capitalize",
+        fontFamily: "'Ropa Sans', sans-serif",
+        fontSize: "1.75rem",
+    },
+});
+
+export default function LandingPage() {
+    const classes = useStyles();
+    const history = useHistory();
+
+    return (
+        <div className={styles.page}>
+            <div className={styles.container}>
```

Comment:
any reason we have two divs?

Reply:
Outer `.page` is full height/width of window & sets blue background colour; inner `.container` is a 80vw container for the landing page elements.